app/Http/Controllers/TwilioController.php

```diff
@@ -2,27 +2,20 @@
 
 namespace App\Http\Controllers;
 
-use App\Http\Requests;
+use App\Http\Requests\TwilioRequest as Request;
```

Comment:
I wonder if it'd be better to instead typehint them as `TwilioRequest` and avoid the alias, so it's clearer where these magical methods are coming from to future devs?